### ISSUE TITLE
ThisAssembly from Nerbank.GitVersioning conflicts with ThisAssembly from "ThisAssembly"

### ISSUE THREAD

**Issue by nathanwinder** [dotnet/Nerdbank.GitVersioning#838]
## Issue Report to "ThisAssembly"
https://github.com/devlooped/ThisAssembly/issues/124

## Describe the Bug
ThisAssembly conflicts with the "ThisAssembly" generated by Nerdbank.GitVersioning
https://github.com/dotnet/Nerdbank.GitVersioning

## Steps to Reproduce

1. Reference both Nerdbank.GitVersioning and ThisAssembly.Project
```
<PackageReference Include="ThisAssembly.Project" Version="1.0.9" Condition="!Exists('packages.config')"/>
<PackageReference Include="Nerdbank.GitVersioning" Condition="!Exists('packages.config')">
  <PrivateAssets>all</PrivateAssets>
  <Version>3.5.113</Version>
</PackageReference>
```

```c#
public class ReproTest
{
  [Fact]
  public void Repro()
  {
    var fileVersion = ThisAssembly.AssemblyFileVersion; // 'ThisAssembly' does not contain a definition for 'AssemblyName'
  }
}
```

## Expected Behavior

ThisAssembly.Project should not prevent Nerbbank.GitVersioning from creating ThisAssembly properties.

**Comment by AArnott** [comment]
Nerdbank.GitVersioning generates ThisAssembly the same regardless of how/where you reference nb.gv. 
To get along with another `ThisAssembly` generator, what would it take? Can you include details of the conflict? I see we already use the `partial` modifier or the class declaration, so I don't know what else we could do. If the properties themselves conflict, maybe you're using two packages that are too similar to be combined in one project.

**Comment by alexrp** [comment]
I suppose this can be closed as of #911?
